app/helpers/application_helper.rb

```diff
@@ -62,4 +62,32 @@ def filetype_icon(attachment)
     html = "<i class='fa fa-file" + fontawesome_filetype + "-o'></i>"
     html.html_safe
   end
+
+  def show_poll(poll_id = nil)
+    render partial: "shared/polls", locals: { poll_id: poll_id }
```

Comment:
I'd check here here also that the poll is from the current site.

If the poll id is wrong I would return `render nothing: true`